cli/configssh.go

```diff
@@ -277,34 +344,59 @@ func configSSH() *cobra.Command {
 				sort.Strings(wc.Hosts)
 				// Write agent configuration.
 				for _, hostname := range wc.Hosts {
-					configOptions := []string{
-						"Host coder." + hostname,
-					}
-					for _, option := range sshConfigOpts.sshOptions {
-						configOptions = append(configOptions, "\t"+option)
-					}
-					configOptions = append(configOptions,
-						"\tHostName coder."+hostname,
-						"\tConnectTimeout=0",
-						"\tStrictHostKeyChecking=no",
+					sshHostname := fmt.Sprintf("%s%s", coderdConfig.HostnamePrefix, hostname)
+					var configOptions sshConfigOptions
+					// Add standard options.
+					err := configOptions.addOptions(
+						"HostName "+sshHostname,
```

Comment:
This value will get passed to the proxy command, don't we need to handle the new prefix in `coder ssh` as well?

Reply:
Oh right, I forgot the proxy command hard-codes the workspace name, my bad.